multiserver/rair_rlanguageserver.json

```diff
@@ -0,0 +1,24 @@
+{
+  "completion": ["rlanguageserver"],

```

Comment:
you should create file `langserver/rlanguageserver.json` if you want build multi-server.